debian/control

```diff
@@ -8,7 +8,7 @@ Vcs-Git: https://github.com/endlessm/eos-phone-home.git
 
 Package: eos-phone-home
 Architecture: all
-Depends: wget, python, ${misc:Depends}
+Depends: wget, python3, python3-requests, ${misc:Depends}
```

Comment:
Yeah - this is a bigger question I suppose. I'm on the fence for it tbh.

@ramcq / @brent: Thoughts?